scripts/amd/gemm/matmul_kernel.py

```diff
@@ -12,6 +12,7 @@ def matmul_kernel(
     stride_bias,
     BLOCK_SIZE_M: tl.constexpr, BLOCK_SIZE_N: tl.constexpr, BLOCK_SIZE_K: tl.constexpr,
     SPLIT_K: tl.constexpr, GROUP_SIZE_M: tl.constexpr, BIAS: tl.constexpr,
+    EVEN_K: tl.constexpr
```

Comment:
does EVEN_K have to put in argument lists ?
can we put in the kernel like below ?
 EVEN_K = K % BLOCK_SIZE_K == 0

Reply:
Since K is not a constexpr, EVEN_K will not be a constexpr if you compute EVEN_K like this. 
We need EVEN_K to be constexpr so that triton compiler can generate different code based on the value of EVEN_K.

Another way is to make K a constexpr. We can try it in the future.